### ISSUE TITLE
Language around resetting passwords is inconsistent about what actually happens

### ISSUE THREAD

**Issue by pinterface** [dreamwidth/dw-free#1780]
Sometimes it claims it will send you your password, other times it says reset.  It should probably use consistent language about resetting the password, since that's what actually happens.

In the welcome e-mail:
> If you need to retrieve your password, you can do so at any time by visiting the following URL: 

Should probably be more like:
> If you've forgotten your password, you can reset it at any time by visiting the following URL:

On the lostinfo page, it says:
> If you've lost your password, enter your account name and, optionally, the email address to which you'd like the password sent.

That should probably say something more like:
> ... to which you'd like to send a password reset code.

or

> ... the e-mail address you'd like to use to reset your password.

Or whatever, if you can come up with less awkward text.

In the "Lost Password" e-mail itself, it says:
> If you didn't ask to have your password emailed to you, don't be alarmed

Which should probably be more along the lines of:
> If you didn't ask to reset your password, don't be alarmed

I've not grepped through the language files, so there may be other instances I'm missing, but that's what I've seen rummaging through the process.

**Comment by rahaeli** [comment]
So, I went to go make the changes on-site and it looks like most of them have already been made (via translation file) in en or en_DW and just didn't get back-ported into the translation strings in the code. Which is good in that the site is mostly not talking about emailing you your password, and bad in that the code and the site are inconsistent. :)

Changes that have been made on-site:

/lostinfo.bml.lostpassword.text: If you've lost your password, enter your account name and, optionally, the email address to which you'd like the password reset token sent.  The email address must be one you've already confirmed and used previously at [[sitename]].  If you leave the email field blank, the password reset email will be sent to your current email address.

This information was requested on the website from [[remoteip]].

/lostinfo_do.bml.lostpasswordmail.part3: If you didn't ask to have a password reset token emailed to you, don't be alarmed - after all, you're the one reading this email, not the person who made the request on the website.  It's possible that the person who made the request simply made a typo in an account name or email address. 

email.newacct5.body has been changed to be more DW-specific, but the version in dw-free should change "retrieve your password" to "reset your password".
